backend/src/main/java/com/coyjiv/isocial/resource/rest/FriendController.java

```diff
@@ -112,6 +112,10 @@ public ResponseEntity<?> availableFriendRequests(@RequestParam(defaultValue = "0
     return ResponseEntity.ok(friendRequests);
   }
 
-
+  @GetMapping("/recommended")
```

Comment:
I think it's better to use "recommendations" name, so it's the same as in PostRestController